### DISCUSSION TITLE
Please help. overrideWithValue method is no longer available on ChangeNotifierProvider class ?

### DISCUSSION THREAD

**Discussion by twittergadungan** [rrousselGit/riverpod#2106] [Q&A] [answered]
Hi, I am trying to update my `flutter_riverpod` from `1.0.4` to the latest version `2.1.3`. but unfortunately I have errors on my widget tests like this https://i.stack.imgur.com/i9fgz.png

it seems that `overrideWithValue` method is no longer available on `ChangeNotifierProvider` class. it seems `overrideWithValue` method is only available on `Provider` class. what should I do to solve this error? please help :)

**Comment by AhmedLSayed9** [comment, marked as answer]
`notifierPovider.overrideWith((ref) => mockNotifier)`
